README.md

```diff
@@ -1,36 +1,32 @@
-Blog App
+##Blog App
```

Comment:
- [ ] It would be better if the README file is properly formatted to look professional, let us consider leave space between our text and symbols. Let us improve the README